jenkins_cli/cli.py

```diff
@@ -31,6 +32,11 @@
                   }
 
 
+# Default constants to parse booleans from .jenkins-cli
+TRUE = ['yes', 'Yes', 'YES', 'True', 'true', 'TRUE', '1']
```

Comment:
I agree, any case scenario this should be documented on both `README.md` and `--help`.

We could document that the user should set `ignore_ssl=yes` and all other cases will be treated as `False` and still have the list. 

If we decide to accept only one option the comparison will be pretty much the same:

List:
```python
cls.ignore_ssl = ignore_ssl or settings_dict.get('ignore_ssl', 'False') in TRUE
```

Single accepted pattern:
```python
cls.ignore_ssl = ignore_ssl or settings_dict.get('ignore_ssl', 'False') == '1'
```

BTW, I was inspired by [ansible](https://github.com/ansible/ansible/blob/devel/lib/ansible/constants.py#L33) when I wrote this list. They pretty much have the same problem, reading config and transforming what it seems to be boolean into python `bool`.